wagtailcolumnblocks/blocks.py

```diff
@@ -7,16 +7,27 @@
 from django.apps import apps
 from django.templatetags.static import static
 from django.utils.html import format_html
-
-from wagtail.core import blocks, hooks
-
-
-@hooks.register('insert_editor_css')
-def editor_css():
-    return format_html(
-        '<link rel="stylesheet" href="{}">',
-        static('wagtailcolumnblocks/columns.css')
-    )
+from wagtail import VERSION as wagtail_version
+
+
+if wagtail_version >= (5, 0):
+    from wagtail import blocks, hooks
+
+    @hooks.register('insert_global_admin_css')
+    def global_admin_css():
```

Comment:
**suggestion**: because this function is the same in both cases, it might be worth having outside the conditional blocks & conditionally decorating it

Reply:
I have a (weak) preference for making blocks like this such that each chunk is standalone, so if you _only_ want to support 5.0 and later, you just delete the version check and the `else` block.

I'm not terribly attached to it though.

Were you thinking one of these two alternatives below? Or something else?
```
if wagtail_version >= (5, 0):
    from wagtail import blocks, hooks
    hook_register_name = 'insert_global_admin_css'

else:
    from wagtail.core import blocks, hooks
    hook_register_name = 'insert_editor_css'


@hooks.register(hook_register_name)
def global_admin_css():
    return format_html(
        '<link rel="stylesheet" href="{}">',
        static('wagtailcolumnblocks/columns.css')
    )
```

or

```
if wagtail_version >= (5, 0):
    from wagtail import blocks, hooks
    hook_decorator_fn = @hooks.register('insert_global_admin_css')

else:
    from wagtail.core import blocks, hooks
    hook_decorator_fn = @hooks.register('insert_editor_css')


@hook_decorator_fn
def global_admin_css():
    return format_html(
        '<link rel="stylesheet" href="{}">',
        static('wagtailcolumnblocks/columns.css')
    )
```